src/MpesaDaraja/Http/C2B/ProcessCustomerPayment.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * Date 14/03/2023
+ *
+ * @author   kelvin mukotso <kelvinmukotso@gmail.com>
+ */
+
+
+namespace Ssiva\MpesaDaraja\Http\C2B;
+
+use Ssiva\MpesaDaraja\Http\AbstractDarajaQuery;
+
+class ProcessCustomerPayment extends AbstractDarajaQuery
+{
+    protected string $endpoint = 'mpesa/c2b/v1/simulate';
+
+    /*
+     * Make a C2B Payment Request
+     */
+    public function submitRequest(array $params = [], string $app = 'default')
+    {
+        // converts the first character of each indexes in a string to uppercase
+        //user Params to include (Amount ,phone number initiating the C2B transaction (Msisdn)
+        // , BillRefNumber  (Account Number N/B for PayBill Only ), ShortCode(optional)
+        $userParams = formatParams($params);
```

Comment:
- $userParams, in this case, could be 2 or 3 depending on whether the CommandID config param is `CustomerPayBillOnline (PayBill )  - CustomerBuyGoodsOnline (Buy Goods )`
   -  CustomerPayBillOnline needs an extra $userParam of BillRefNumber

- The $configParams again are only 2
```php
        $commandId = configStore()->get('mpesa.c2b.default_command_id');
        $shortCode = configStore()->get('mpesa.c2b.short_code')

        $configParams = [
            'CommandID' => $commandId,
            'ShortCode' => $shortCode,
        ];
```